BASE/src/Microsoft.ApplicationInsights/Metrics/Implementation/AutocollectedMetricsExtraction/MetricTerms.cs

```diff
@@ -4,7 +4,8 @@
 
     internal static class MetricTerms
     {
-        private const string MetricPropertiesNamePrefix = "_MS";
+        internal const string DimensionCapFallbackValue = "DIMENSION_CAPPED";
```

Comment:
This PR does not seem to consume `DimensionCapFallbackValue`, should we update `dimensionCapValue: ` definition above?

Reply:
This was not part of this PR :D it was from my upcoming PR with standard metric. it just got accidentaly spilled into this..